hazelcast/reactor.py

```diff
@@ -228,17 +433,21 @@ def handle_read(self):
             reader.process()
 
     def handle_write(self):
-        with self._write_lock:
```

Comment:
Leaving a comment for L426 `data = self.recv(self.read_buffer_size)` here. Don't we need to have a `data == b''` check for the closed connection case after this call?

Reply:
This is handled by the asyncore internally. If the received data has 0 length, the connection on our side will also be closed.

Reply:
I see, thanks.